script.js

```diff
@@ -1,4 +1,275 @@
+        player2ConcatenatedNumber
+      );
+    }
+  }
+};
+
+//Create a function that outputs the notification statement after the players' have rolled the dice on what their dice number is.
+var statementForDiceRolled = function () {
+  //Check if only player 1 has rolled
+  if (hasPlayer1Rolled == 1 && hasPlayer2Rolled == 0) {
+    return `Welcome Player 1. <br><br>You rolled ${player1DiceRolls[0]} for Dice 1 and ${player1DiceRolls[1]} for Dice 2. <br><br>Choose the order of the dice. <br>Input "dice 1" if you wish for Dice 1's number to be in front of Dice 2's number. <br>Input "dice 2" if you wish for Dice 2's number to be in front of Dice 1's number.`;
+  }
+
+  //Check if player 2 has rolled, after player 1 has rolled
+  if (hasPlayer1Rolled == 1 && hasPlayer2Rolled == 1) {
+    return `Welcome Player 2. <br><br>You rolled ${player2DiceRolls[0]} for Dice 1 and ${player2DiceRolls[1]} for Dice 2. <br><br>Choose the order of the dice. <br>Input "dice 1" if you wish for Dice 1's number to be in front of Dice 2's number. <br>Input "dice 2" if you wish for Dice 2's number to be in front of Dice 1's number.`;
+  }
+};
+
+//Create a function that outputs the notification statement after the players' have chosen which dice to concatenate and return the concatenated number
+var statementForConcatenation = function (userInput) {
+  //Check if only player 1 has rolled
+  if (hasPlayer1Rolled == 1 && hasPlayer2Rolled == 0) {
+    return `Player 1, you chose ${capitalizer(
+      userInput
+    )} first. <br>Your number is ${player1ConcatenatedNumber}. <br>It is now Player 2's turn.`;
+  }
+
+  //Check if player 2 has rolled, after player 1 has rolled
+  if (hasPlayer1Rolled == 1 && hasPlayer2Rolled == 1) {
+    return `Player 2, you chose ${capitalizer(
+      userInput
+    )} first. <br>Your number is ${player2ConcatenatedNumber}. <br>It is now time to see who won! Type in "compare" to see who won.`;
+  }
+};
+
+//Create a function that compares the two numbers and output a who wins statement.
+var comparingWinnerStatement = function () {
+  //Create a variable that is a statement to let the players know how to reset the game
+  var howToReset = `Please input "reset" if you wish to reset the game and play again.`;
+
+  //Check if there is a tie
+  if (player1ConcatenatedNumber == player2ConcatenatedNumber) {
+    return `Both of your numbers are ${player1ConcatenatedNumber}. <br><br>It's a tie! <br><br>${howToReset}`;
+  }
+
+  //Check if player 1 won
+  if (player1ConcatenatedNumber > player2ConcatenatedNumber) {
+    return `Player 1's number is ${player1ConcatenatedNumber}. <br>Player 2's number is ${player2ConcatenatedNumber}. <br><br>Player 1 won! <br><br>${howToReset}`;
+  }
+
+  //Check if player 2 won
+  if (player2ConcatenatedNumber > player1ConcatenatedNumber) {
+    return `Player 1's number is ${player1ConcatenatedNumber}. <br>Player 2's number is ${player2ConcatenatedNumber}. <br><br>Player 2 won! <br><br>${howToReset}`;
+  }
+};
```

Comment:
nice use of early return here to organise the code. because of this use of early return, we do not need the last if in line 166, so this can be just 

```
if (player1ConcatenatedNumber == player2ConcatenatedNumber) {
    return `Both of your numbers are ${player1ConcatenatedNumber}. <br><br>It's a tie! <br><br>${howToReset}`;
  }

  //Check if player 1 won
  if (player1ConcatenatedNumber > player2ConcatenatedNumber) {
    return `Player 1's number is ${player1ConcatenatedNumber}. <br>Player 2's number is ${player2ConcatenatedNumber}. <br><br>Player 1 won! <br><br>${howToReset}`;
  }

  //Check if player 2 won
    return `Player 1's number is ${player1ConcatenatedNumber}. <br>Player 2's number is ${player2ConcatenatedNumber}. <br><br>Player 2 won! <br><br>${howToReset}`;
```
The way the code above works is: if the first condition is met return early, otherwise it carries on to the second condition, again if it is met then we return early. Otherwise we carry on to the last part which does not need to be in an if statement, this serves as our 'default' case if we didn't return early